src/components/Button.js

```diff
@@ -1,12 +1,152 @@
 import React from "react";
+import styled from 'styled-components';
 
-const Button = props => (
-  <button
-    onClick={props.onClick}
-    className={props.className}
-    style={props.style}
-  >
+
+//Here starts our specific button styles: 
+
+
+/* primary buttons styling */
+const btnPrimary = {
+  name :"btnPrimary",
+  color: "#ffffff",
+  background: "#2196f3"
+};
+
+const btnPrimaryOutlined = {
+  name :"btnPrimaryOutlined",
+   color: "#2196f3",
+  background: "#ffffff",
+  border: "2px solid #03a9f4"
+};
+
+const btnPrimaryLight = {
+  name : "btnPrimaryLight",
+  color: "#2196f3",
+  background: "#b3e5fc"
+};
+
+const btnPrimaryGrayShadow ={
+  name :"btnPrimaryGrayShadow",
+  color: "#2196f3",
+  background: "#ffffff",
+  boxShadow: "0px 1px 10px rgba(0, 0, 0, 0.2), 0px 4px 5px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14)"
+};
+const btnPrimaryShadow = {
+  name:"btnPrimaryShadow",
+  color: "#2196f3",
+  background: "#ffffff",
+  boxShadow: "0px 2px 4px #03A9F4"
+
+};
+
+
+/* secondary buttons styling */
+const btnSecondary = {
+  name : "btnSecondary",
```

Comment:
What are you using name for?

Reply:
I added this property to enable us decide which button styling we will apply according to the type property "variant"